Source/WebCore/accessibility/AccessibilityScrollView.h

```diff
@@ -85,11 +88,11 @@ class AccessibilityScrollView final : public AccessibilityObject {
     AccessibilityScrollbar* addChildScrollbar(Scrollbar*);
     void removeChildScrollbar(AccessibilityObject*);
 
+    bool m_childrenDirty;
     SingleThreadWeakPtr<ScrollView> m_scrollView;
     WeakPtr<HTMLFrameOwnerElement, WeakPtrImplWithEventTargetData> m_frameOwnerElement;
     RefPtr<AccessibilityObject> m_horizontalScrollbar;
     RefPtr<AccessibilityObject> m_verticalScrollbar;
-    bool m_childrenDirty;
```

Comment:
Moving this field makes AccessibilityScrollView instances 8 bytes smaller by reducing unnecessary padding.